src/modules/NetworkController/public/Start-SdnNetworkControllerCertificateUpdate.ps1

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Start-SdnNetworkControllerCertificateUpdate {
+    <#
+    .SYNOPSIS
+        Start the Network Controller Certificate Update.
+    .DESCRIPTION
+        Start the Network Controller Certificate Update.
+        This will use the latest issued certificate on each of the NC VMs to replace existing certificates. Ensure below before execute this command:
+        - NC Rest Certificate and NC Node certificate created on each NC and trusted.
+        - "Network Service" account have read access to the private file of the new certificates.
+        - NC Rest Certificate need to be trusted by all SLB MUX VMs and SDN Hosts.
+
+        For Self-Signed Certificate. This can also be created by 'New-NetworkControllerCertificate'. To get more details, run 'Get-Help New-NetworkControllerCertificate'
+
+        About SDN Certificate Requirement:
+        https://docs.microsoft.com/en-us/windows-server/networking/sdn/security/sdn-manage-certs
+
+    .PARAMETER CertRotateConfig
+        The Config generated by New-SdnCertificateRotationConfig to include NC REST certificate thumbprint and node certificate thumbprint.
+    .EXAMPLE
+        Start-NetworkControllerCertificateUpdate -NetworkController nc01
+    #>
+
+    param (
+        [Parameter(Mandatory = $true)]
+        [hashtable]
+        $CertRotateConfig,
+        [System.Management.Automation.PSCredential]
+        [System.Management.Automation.Credential()]
+        $Credential = [System.Management.Automation.PSCredential]::Empty,
+        [System.Management.Automation.PSCredential]
+        [System.Management.Automation.Credential()]
+        $NcRestCredential = [System.Management.Automation.PSCredential]::Empty
+    )
+
+    # ensure that the module is running as local administrator
+    $elevated = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
+    if (-NOT $elevated) {
+        throw New-Object System.Exception("This function requires elevated permissions. Run PowerShell as an Administrator and import the module again.")
+    }
+
+    $config = Get-SdnRoleConfiguration -Role 'NetworkController'
+    $confirmFeatures = Confirm-RequiredFeaturesInstalled -Name $config.windowsFeature
+    if (-NOT ($confirmFeatures)) {
+        throw New-Object System.NotSupportedException("The current machine is not a NetworkController, run this on NetworkController.")
+    }
+
+    $NcUpdateFolder = "$(Get-WorkingDirectory)\NcCertUpdate_{0}" -f (Get-FormattedDateTimeUTC)
+    $ManifestFolder = "$NcUpdateFolder\manifest"
+    $ManifestFolderNew = "$NcUpdateFolder\manifest_new"
+
+    $result = Test-NetworkControllerRemoteAccess -Credential $Credential
```

Comment:
The purpose was to validate remote access for each of the NC, in case failed we didn't continue with cert rotation to break anything. This should have been handled in cert config validation now, if there is any issue with ps remoting the cert rotation should have been terminated. So removed this test now.